index.html

```diff
@@ -6,6 +6,84 @@
   <title>Document</title>
 </head>
 <body>
+  <header class="header-main">
+    <h1>
+      DEVICE
+    </h1>
+    <nav>
+      <ul class="site-navigation">
+        <li><a href="site-search.html"><img src="svg/search.svg">Поиск</a></li>
+        <li><a href="site-user.html"><img src="svg/user.svg">Войти</a></li>
+        <li><a href="site-compare.html"><img src="svg/compare.svg">Сравнить</a></li>
+        <li><a href="site-cart.html"><img src="svg/cart.svg">Корзина</a></li>
+        <li><a href="site-plus.html"><img src="svg/plus.svg">Каталог товаров</a></li>
+        <li><a href="site-search.html"><img src="svg/#.svg">Доставка</a></li>
+        <li><a href="site-search.html"><img src="svg/#.svg">Гарантия</a></li>
+        <li><a href="site-search.html"><img src="svg/#.svg">Контакты</a></li>
```

Comment:
У изображений должен быть атрибут `alt=""`, указывающий информацию о изображении для скринридеров, и на случай, если картинка не пришла с сервера

```suggestion
        <li><a href="site-search.html"><img src="#" alt="Иконка поиска">Поиск</a></li>
        <li><a href="site-user.html"><img src="#" alt="Иконка входа">Войти</a></li>
        <li><a href="site-compare.html"><img src="#" alt="Иконка сравнить">Сравнить</a></li>
        <li><a href="site-cart.html"><img src="#" alt="Иконка корзины">Корзина</a></li>
        <li><a href="site-plus.html"><img src="#" alt="Плюсик">Каталог товаров</a></li>
        <li><a href="site-search.html">Доставка</a></li>
        <li><a href="site-search.html">Гарантия</a></li>
        <li><a href="site-search.html">Контакты</a></li>
```